deployments/helm/templates/deployment.yaml

```diff
@@ -57,13 +57,18 @@ spec:
             cpu: {{ .Values.deployment.resources.requests.cpu | quote}}
             memory: {{ .Values.deployment.resources.requests.memory }}
         {{- if not .Values.backup.enabled }}
+        env:
+        - name: RUST_LOG
+          value: papyrus=DEBUG
```

Comment:
It would be easier to place this key-value pair in the configmap which it's content is exported as env-vars https://github.com/starkware-libs/papyrus/blob/98be514c8ff7c9c44d95c406598553da73554b18/deployments/helm/templates/deployment.yaml#L75